datafusion/physical-plan/src/values.rs

```diff
@@ -34,6 +34,7 @@ use datafusion_execution::TaskContext;
 use datafusion_physical_expr::EquivalenceProperties;
 
 /// Execution plan for values list based relation (produces constant rows)
+#[deprecated(since = "44.0.0", note = "Use `MemoryExec::try_new_as_values` instead")]
```

Comment:
Since we have already released 44, I think the correct version is the next unreleased one, 45

```suggestion
#[deprecated(since = "45.0.0", note = "Use `MemoryExec::try_new_as_values` instead")]
```